### PR TITLE
Events

### PR DESCRIPTION
**Pull request by SelmaNdapanda**
## In this pull request, I performed the following:

- I copied the files calculate.js and operate.js into a logic/ directory in my project.
- installed big.js and used it as a dependency
- Imported the files in my Calculator component.
- Implemented the event handlers I need to use the math logic from calculate.js and operate.js in my React component.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,71 +1,60 @@
...
+            <button type="button" onClick={this.clickHandler} className="left-btns" name="5"> 5 </button>
+            <button type="button" onClick={this.clickHandler} className="left-btns" name="6"> 6 </button>
+            <button type="button" onClick={this.clickHandler} className="right-btns" name="-"> &ndash; </button>
+            <button type="button" onClick={this.clickHandler} className="left-btns" name="1"> 1 </button>
+            <button type="button" onClick={this.clickHandler} className="left-btns" name="2"> 2 </button>
+            <button type="button" onClick={this.clickHandler} className="left-btns" name="3"> 3 </button>
+            <button type="button" onClick={this.clickHandler} className="right-btns" name="+"> + </button>
+            <button type="button" onClick={this.clickHandler} className=" left-btns zero" name="0"> 0 </button>
+            <button type="button" onClick={this.clickHandler} className="left-btns" name="."> . </button>
+            <button type="button" onClick={this.clickHandler} className="right-btns" name="="> = </button>
```
Comment (DammyShittu) [inline src/components/Calculator.js:50]: - [x] When I set up a calculation and click this equal sign to perform the calculation the app crashes and displays nothing. This is the error message I get from the console. Kindly look into that and fix it.

  ryrw

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @SelmaNdapanda,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (SelmaNdapanda) [reply src/components/Calculator.js:50]: Okay thank you. I found a solution

**Review by descholar-ceo — APPROVED** [review]
Hi @SelmaNdapanda 🤝 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Implemented the requested changes ✔️ 
👍🏽 Great communication ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**